native/acter/src/api/spaces.rs

```diff
@@ -602,22 +602,13 @@ impl Client {
             .await?
     }
 
-    pub async fn get_space(&self, alias_or_id: String) -> Result<Space> {
-        if let Ok(room_id) = OwnedRoomId::try_from(alias_or_id.clone()) {
+    pub async fn get_space(&self, room_id: String) -> Result<Space> {
+        if let Ok(room_id) = OwnedRoomId::try_from(room_id.clone()) {
             self.get_room(&room_id)
                 .context("Room not found")
                 .map(|room| Space::new(self.clone(), Room { room }))
-        } else if let Ok(alias_id) = OwnedRoomAliasId::try_from(alias_or_id) {
-            for space in self.spaces().await?.into_iter() {
-                if let Some(space_alias) = space.inner.room.canonical_alias() {
-                    if space_alias == alias_id {
-                        return Ok(space);
-                    }
-                }
-            }
-            bail!("Room with alias not found");
         } else {
-            bail!("Neither roomId nor alias provided");
+            bail!("roomId not provided");
```

Comment:
I see what you did here... but wonder ... this is async ... shouldn't we instead have that `resolve_alias`-call be a fallback if we can't find the room here? I'd very much prefer that

Reply:
Fixed